README.md

```diff
@@ -1,64 +1,228 @@
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Installation
+
+_Now that the environment has been set up and configured to properly compile and run the project, the next step is to install and configure the project locally on your system._
+<!-- UPDATE -->
+1. Clone the repository
+   ```sh
+   git clone https://github.com/metakgp/gyft
+   ```
+2. Change current directory to the project directory
+   ```sh
+   cd gyft
+   ```
+3. Install the dependencies
+   ```sh
+   pip install requirements.txt
+   ```
+4. Run the script
+   ```sh
+   python3 gyft.py
+   ```
     - Optional flags: 
     	- `--output` or `o`: Output file (`.ics`) to which the ICS file will be written. **Default:** `timetable.ics`
-    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False`
+    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False` 
 
 4. Enter your roll number, password, security answer and OTP (if required) when prompted
-5. Now you can choose between 2 options, which will be prompted to you:
+5. Your timetable will be saved in the file specified by the `--input` flag (**default:** `data.txt`)
+6. Now you can choose between 2 options, which will be prompted to you:
    - #### Add to Google Calendar:
      - For this, you have to get your `credentials.json` and save it as `client_secret.json` to the current directory by following the Step 1 from [here](https://developers.google.com/google-apps/calendar/quickstart/python#step_1_turn_on_the_api_name)
      - Also note that adding to Google Calendar requires an Internet connection
   
    - #### Generate ICS file:
      - This will generate an ICS file which you can import into any calendar application.
      - For importing to Google Calendar, follow the instructions given [here](https://support.google.com/calendar/answer/37118?hl=en).
-  
-> **Note** 
-> - You can also use the web application to get your ICS file
-> - Go to the [GYFT WebApp](https://gyft.metakgp.org/)
-> - Enter your roll number and get the security question
-> - Once the security question is fetched, enter your credentials and save the ICS file
-> - Source at [metakgp/gyft-serve](https://github.com/metakgp/gyft-serve)
-
-#### Miscellaneous 
-To delete all the recurring events added by the CLI or GYFT Web Application (uses property of recurrence to identify events to delete), you can run `gyft.py` with the `--del-events` flag:
-```sh
-python3 gyft.py --del-events
-```
+
+<!-- <p align="right">(<a href="#top">back to top</a>)</p> -->
+
+
+<!-- USAGE EXAMPLES -->
+<!-- ## Usage -->
+<!-- UPDATE -->
+<!-- <div align="center">
+  <a href="https://github.com/proffapt/PROJECT_NAME">
+    <img width="80%" alt="image" src="https://user-images.githubusercontent.com/86282911/206632640-40dc440e-5ef3-4893-be48-618f2bd85f37.png">
+  </a>
+</div> -->
```

Comment:
Add usage examples for deleting the events and any other tags if there are - I lost track of the progress in here